js/geo/coordinate.js

```diff
@@ -15,6 +15,7 @@ function Coordinate(column, row, zoom) {
     this.column = column;
     this.row = row;
     this.zoom = zoom;
+    this.posMatrix = null;
```

Comment:
> calculatePosMatrix should probably not be a Painter method. Where should it go?

It could always be a standalone module.

> Caching in `calculatePosMatrix` would be much more complicated technically to make it right. 

What if we clear the cache every time `transform` changes?